README.md

```diff
@@ -124,6 +124,8 @@ spec:
   # when enabled, and if Istio is installed on the Kubernetes environment, then the Operator will create the corresponding
   # VirtualService and ServiceEntries objects to allow access to the Dynatrace cluster from the agent.
   #enableIstio: false
+  # DNS Policy for OneAgent pods (optional.) Empty for default, other options: ClusterFirstWithHostNet, etc.
```

Comment:
`other options: ClusterFirstWithHostNet, etc`

"etc" is not very helpful - either please link a documentation link or describe this with available options. I assume not everyone would memorize the available DNS Policies? :grin: